openassessment/xblock/apis/assessments/self_assessment_api.py

```diff
@@ -0,0 +1,64 @@
+from openassessment.assessment.api import self as self_api
+from submissions import api as submission_api
+
+from openassessment.xblock.utils.data_conversion import create_submission_dict
+from openassessment.xblock.apis.step_data_api import StepDataAPI
+
+
+class SelfAssessmentAPI(StepDataAPI):
```

Comment:
Are you not getting lint errors that these are missing docstrings?

Reply:
@jansenk , those are disabled by our `pylint` config... though we should probably have them, right @muselesscreator?